Description: debianize dump-utmp.8 manpage
Author: Daniel Baumann <daniel@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/dump-utmp.8
+++ b/dump-utmp.8
@@ -1,29 +1,65 @@
-.\" Copyright (C) 2009 Red Hat, Inc. All Rights Reserved.
-.\" Written by Ivana Hutarova Varekova <varekova@redhat.com>
-.TH DUMP_UTMP 8 "2009 December 2" "Linux accounting system"
+.TH DUMP-UTMP 8 "2006-04-22" "6.4pre1" "GNU Accounting Utilities"
 
 .SH NAME
-dump\-utmp \- print a utmp file in human-readable format
+dump-utmp \- print an utmp file in human-readable format.
 
 .SH SYNOPSIS
-\fBdump-utmp\fR [\fB-hrR\fR] [\fB-n\fR <\fIrecs\fR>] <\fBfiles\fR>
-      [\fB--num\fR <\fIrecs\fR>] [\fB--raw\fR] [\fB--reverse\fR] [\fB--help\fR]
+.B dump-acct
+.RB [\| \-r \||\| \-\-reverse \|]
+.RB [\| \-R \||\| \-\-raw \|]
+.RB [\| \-n \||\| \-\-num
+.IR recs \|]
+.RB [\| \-h \||\| \-\-help \|]
+.RB [\|
+.IR files \|]
+
+.SH DESCRIPTION
+.B dump-acct
+.I filename
+prints a list of all logins. This list is usually written in
+.IR /var/log/wtmp .
+.LP
+All fields are separated by vertical line. Fields are:
+.IR "user name" ,
+.IR tty ,
+.IR type ,
+.IR id ,
+.IR pid ,
+.IR hostaddr ,
+.IR host ,
+.IR time .
 
 .SH OPTIONS
-The following options are supported:
 .TP
-.B -h, --help
-Print a help message and the default location of the process accounting file
-and exit.
+.B \-h, \-\-help
+Prints the usage string and default locations of system files to standard output
+and exits.
+.TP
+.B \-n, \-\-num recs
+Number of lines to print.
 .TP
-.B -r, --reverse
-Print the output in reverse order.
+.B \-r, \-\-reverse
+Start printing from last records.
 .TP
-.B -R, --raw
-The records will be printed without any parsing.
+.B \-R, \-\-raw
+Print raw records, not human-readable.
+
+.SH FILES
 .TP
-.B -n, --num NUMRECS
-Display only the first NUMRECS number of records.
+.I wtmp
+The system wide login record file. See
+.BR wtmp (5)
+for further details.
 
 .SH SEE ALSO
-accton (8), lastcomm (1), utmp (5)
+.BR login (1),
+.BR wtmp (5),
+.BR init (8),
+.BR sa (8).
+
+.SH AUTHOR
+The GNU accounting utilities were written by Noel Cragg <noel@gnu.ai.mit.edu>.
+.PP
+This manual page was written by Ognyan Kulev <ogi@fmi.uni-sofia.bg> and updated
+by Daniel Baumann <daniel@debian.org> for the Debian project (but may be used by
+others).
